sdks/go/pkg/beam/core/graph/coder/coder.go

```diff
@@ -394,6 +395,19 @@ func NewKV(components []*Coder) *Coder {
 	}
 }
 
+func NewN(component *Coder) *Coder {
+	checkCoderNotNil(component, "Nullable")
+	return &Coder{
+		Kind:       Nullable,
+		T:          typex.New(typex.NullableType, component.T),
+		Components: []*Coder{component},
+	}
+}
```

Comment:
I think you may need to update https://github.com/apache/beam/blob/7e896ba429fdad0b54448409cff95bdfd1d03e92/sdks/go/pkg/beam/core/runtime/exec/coder.go#L67 (and `MakeElementDecoder`) as well to correctly plumb this through

Reply:
Pls address this.

Reply:
Added